release_notes/ocp-4-16-release-notes.adoc

```diff
@@ -0,0 +1,1068 @@
+
+// Added in 4.14. Language came directly from Kirsten Newcomer.
+{product-title} is designed for FIPS. When running {op-system-base-full} or {op-system-first} booted in FIPS mode, {product-title} core components use the {op-system-base} cryptographic libraries that have been submitted to NIST for FIPS 140-2/140-3 Validation on only the `x86_64`, `ppc64le`, and `s390x` architectures.
+
+For more information about the NIST validation program, see link:https://csrc.nist.gov/Projects/cryptographic-module-validation-program/validated-modules[Cryptographic Module Validation Program]. For the latest NIST status for the individual versions of {op-system-base} cryptographic libraries that have been submitted for validation, see link:https://access.redhat.com/articles/2918071#fips-140-2-and-fips-140-3-2[Compliance Activities and Government Standards].
+
+[id="ocp-4-16-add-on-support-status"]
+== {product-title} layered and dependent component support and compatibility
+
+The scope of support for layered and dependent components of {product-title} changes independently of the {product-title} version. To determine the current support status and compatibility for an add-on, refer to its release notes. For more information, see the link:https://access.redhat.com/support/policy/updates/openshift[Red Hat {product-title} Life Cycle Policy].
+
+[id="ocp-4-16-new-features-and-enhancements"]
+== New features and enhancements
+
+This release adds improvements related to the following components and concepts.
+
+[id="ocp-4-16-rhcos"]
+=== {op-system-first}
+
+==== {op-system} now uses {op-system-base} 9.4
+
+{op-system} now uses {op-system-base-full} 9.4 packages in {product-title} {product-version}. These packages ensure that your {product-title} instances receive the latest fixes, features, enhancements, hardware support, and driver updates. As an Extended Update Support (EUS) release, {product-title} 4.14 is excluded from this change and will continue to use {op-system-base} 9.2 EUS packages for the entirety of its lifecycle.
+
+[id="ocp-4-16-installation-and-update"]
+=== Installation and update
+
+[id="ocp-4-16-web-console"]
+=== Web console
+
+[id="ocp-4-16-administrator-perspective"]
+==== Administrator perspective
+
+This release introduces the following updates to the *Administrator* perspective of the web console:
+//Are there any for 4.16
+
+[id="ocp-4-16-developer-perspective"]
+==== Developer Perspective
+
+This release introduces the following updates to the *Developer* perspective of the web console:
+//* are there any for 4.16?
+
+[id="ocp-4-16-openshift-cli"]
+=== OpenShift CLI (oc)
+
+[id="ocp-4-16-ibm-z"]
+=== {ibm-z-title} and {ibm-linuxone-title}
+
+[id="ocp-4-16-ibm-power"]
+=== {ibm-power-title}
+
+[id="ocp-4-16-auth"]
+=== Authentication and authorization
+
+[id="ocp-4-16-networking"]
+=== Networking
+
+SHA1 certificates are no longer supported for use with HaProxy. Routes using SHA1 certificates will be rejected.
```

Comment:
I wonder if it's better to be super clear that both new routes you created, and more importantly, the routes you already have, will stop functioning. Someone may think "oh maybe this just means new routes so my cluster is okay".

```suggestion
SHA1 certificates are no longer supported for use with HaProxy. Both existing and new routes using SHA1 certificates in {product-version} will be rejected and no longer function.
```